torch/_inductor/config.py

```diff
@@ -398,6 +398,7 @@ def use_autoheuristic(name: str) -> bool:
 # smaller threshold
 realize_reads_threshold = 4
 realize_opcount_threshold = 30
+realize_opusers_threshold = 5
```

Comment:
Unused?

Reply:
Sorry, this is a mistake. Will be used in `users > 5` in should_realize_on_cpu.

Reply:
Fixed.